kpet/run.py

```diff
@@ -32,8 +44,9 @@ def __init__(self, suite, cases):
         """
```

Comment:
Oh, and please update the documentation to reflect the type of `cases` argument.